src/System.Runtime.Analyzers/Core/SpecifyStringComparison.cs

```diff
@@ -24,15 +29,117 @@ public abstract class SpecifyStringComparisonAnalyzer : DiagnosticAnalyzer
                                                                              s_localizableMessage,
                                                                              DiagnosticCategory.Globalization,
                                                                              DiagnosticSeverity.Warning,
-                                                                             isEnabledByDefault: false,
+                                                                             isEnabledByDefault: true,
                                                                              description: s_localizableDescription,
-                                                                             helpLinkUri: null,     // TODO: add MSDN url
+                                                                             helpLinkUri: "https://msdn.microsoft.com/en-us/library/bb386080.aspx",
                                                                              customTags: WellKnownDiagnosticTags.Telemetry);
 
         public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics => ImmutableArray.Create(Rule);
 
         public override void Initialize(AnalysisContext analysisContext)
         {
+            analysisContext.RegisterCompilationStartAction(csaContext =>
+            {
+                var stringComparisonType = csaContext.Compilation.GetTypeByMetadataName("System.StringComparison");
+                var stringType = csaContext.Compilation.GetSpecialType(SpecialType.System_String);
+                var currentCulture = stringComparisonType.GetMembers("CurrentCulture").SingleOrDefault();
```

Comment:
Using `SingleOrDefault` will cause an exception if there are multiple definitions found (for example a custom build mscorlib or a user defined type in source which defines multiple members with same name). Its generally better to use `.OfType<IXXXSymbol>().FirstOrDefault()`.